assets/files/etc/sysconfig/network-scripts/ifcfg-ens4

```diff
@@ -0,0 +1,5 @@
+DEVICE=ens4
```

Comment:
This file should be a template that is rendered by the dispatcher script once it learns which interface is the right one.

Reply:
switched to a template file created in the dispatcher script

Reply:
@celebdor @karimb could you give a top level review of what you attempt to do in this change?

If find it odd that you look for the default route (set by dhcp?) and attach a bridge to it, but without dhcp? Why restart NM? Why do it within the dispatcher script? And why without any care to the args passed by NM to it?

If expect us to know the nic name in advance, and simply drop here the ifcfg files for it and it's bridge, with dhcp enabled on the bridge.